deltacat/tests/compactor/test_compact_partition_params.py

```diff
@@ -0,0 +1,204 @@
+import json
+
+import pytest
+
+
+@pytest.fixture
+def valid_compact_partition_params():
```

Comment:
We have been using unittest and an unit test class to test all the methods. It will provide a consistent test report.